integrationTests/state/stateTrieSync_test.go

```diff
@@ -40,21 +41,23 @@ func TestNode_RequestInterceptTrieNodesWithMessenger(t *testing.T) {
 
 	time.Sleep(integrationTests.SyncDelay)
 
-	_ = nResolver.StateTrie.Update([]byte("doe"), []byte("reindeer"))
-	_ = nResolver.StateTrie.Update([]byte("dog"), []byte("puppy"))
-	_ = nResolver.StateTrie.Update([]byte("dogglesworth"), []byte("cat"))
-	_ = nResolver.StateTrie.Commit()
-	rootHash, _ := nResolver.StateTrie.Root()
+	resolverTrie := nResolver.TrieContainer.Get([]byte(factory2.UserAccountTrie))
+	_ = resolverTrie.Update([]byte("doe"), []byte("reindeer"))
+	_ = resolverTrie.Update([]byte("dog"), []byte("puppy"))
+	_ = resolverTrie.Update([]byte("dogglesworth"), []byte("cat"))
+	_ = resolverTrie.Commit()
+	rootHash, _ := resolverTrie.Root()
 
-	nilRootHash, _ := nRequester.StateTrie.Root()
-	trieNodesResolver, _ := nRequester.ResolverFinder.IntraShardResolver(factory.TrieNodesTopic)
+	requesterTrie := nRequester.TrieContainer.Get([]byte(factory2.PeerAccountTrie))
```

Comment:
`UserAccountTrie` instead of `PeerAccountTrie`.

Reply:
done.